### PR TITLE
Unify rosa localzones macnine pool config with ocp

### PR DESCRIPTION
**Pull request by Phaow**
### Unify rosa localzones macnine pool config with ocp
-  In OCP CI all the aws local zones CI config all use the edge label,  we'd better unify the label/role (DEFAULT_MACHINE_POOL_LABELS: "node-role.kubernetes.io/worker") setting keep the same with OCP.

After the change we could see the edge node config keep consistent with OCP (the localzone node has the edge role and taints by default)
```console
$ omc get no
NAME                           STATUS   ROLES                  AGE   VERSION
ip-10-0-132-168.ec2.internal   Ready    edge,worker            4h    v1.27.6+f67aeb3
ip-10-0-48-52.ec2.internal     Ready    infra,worker           5h    v1.27.6+f67aeb3
ip-10-0-56-84.ec2.internal     Ready    control-plane,master   5h    v1.27.6+f67aeb3
ip-10-0-57-184.ec2.internal    Ready    worker                 5h    v1.27.6+f67aeb3
ip-10-0-68-26.ec2.internal     Ready    infra,worker           5h    v1.27.6+f67aeb3
ip-10-0-71-164.ec2.internal    Ready    control-plane,master   5h    v1.27.6+f67aeb3
ip-10-0-77-210.ec2.internal    Ready    worker                 5h    v1.27.6+f67aeb3
ip-10-0-90-125.ec2.internal    Ready    control-plane,master   5h    v1.27.6+f67aeb3
ip-10-0-92-128.ec2.internal    Ready    worker                 5h    v1.27.6+f67aeb3
ip-10-0-94-130.ec2.internal    Ready    infra,worker           5h    v1.27.6+f67aeb3
$ omc get no/ip-10-0-132-168.ec2.internal -ojsonpath='{.spec.taints}'|jq .
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/edge"
  }
]
$ omc get no -ojsonpath='{.items[*].spec.taints}'|jq .
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/edge"
  }
]
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/infra"
  }
]
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/master"
  }
]
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/infra"
  }
]
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/master"
  }
]
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/master"
  }
]
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/infra"
  }
]
```

### CODE REVIEW

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
Hi @yasun1 @xueli181114, could you please take a look when you get a chance? Thanks in advance!

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-sts-advanced-f7

File: ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh [hunk 1]
```diff
@@ -77,6 +77,9 @@ fi
 LOCAL_ZONE_SWITCH=""
 if [[ "$LOCAL_ZONE" == "true" ]]; then
   LOCAL_ZONE_SWITCH=""
+  # Unify rosa localzones macnine pool config with ocp
+  LABELS="node-role.kubernetes.io/edge="
+  TAINTS="node-role.kubernetes.io/edge=:NoSchedule"
```
Comment (yasun1) [inline ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh:82]: You need to add it like this to not overwrite the customized label/taint:
`LABELS="${LABELS},node-role.kubernetes.io/edge="`

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-sts-advanced-f7

File: ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh (hunk 1 above)
Comment (Phaow) [reply ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh]: Thanks! Addressed.

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-rosa-sts-advanced-f7

**Comment by Phaow** [comment]
Hi @yasun1 , the rehearse results looks good and the config is as expected consistent with ocp. Could you please help take a look again when you get a chance? Thanks!

**Pull request merged**